Formula/mumps.rb

```diff
@@ -0,0 +1,201 @@
+class Mumps < Formula
+  desc "Parallel Sparse Direct Solver"
+  homepage "http://mumps-solver.org"
+  url "http://mumps.enseeiht.fr/MUMPS_5.1.2.tar.gz"
+  sha256 "eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9"
+
+  depends_on :mpi => [:cc, :cxx, :f90, :recommended]
+  depends_on :fortran
+  depends_on "scalapack"
+  depends_on "metis"    => :optional if build.without? "mpi"
+  depends_on "parmetis" => :optional if build.with? "mpi"
+  depends_on "scotch5"  => :optional
+  depends_on "scotch"   => :optional
+
+  resource "mumps_simple" do
+    url "https://github.com/dpo/mumps_simple/archive/v0.4.tar.gz"
+    sha256 "87d1fc87eb04cfa1cba0ca0a18f051b348a93b0b2c2e97279b23994664ee437e"
+  end
+
+  def install
+    make_args = ["RANLIB=echo"]
+
+    shlibs_args = ["LIBEXT=.dylib",
+                   "AR=#{ENV["FC"]} -dynamiclib -Wl,-install_name -Wl,#{lib}/$(notdir $@) -undefined dynamic_lookup -o "]
```

Comment:
Why this needed? It looks like an upstream bug and/or total lack of macOS support.

Reply:
These lines were in the original formula on homebrew-science, not quite sur why that's here. I'll try to clean it up a bit.